packages/terra-application-utility/CHANGELOG.md

```diff
@@ -4,3 +4,4 @@ ChangeLog
 Unreleased
 -----------------
 * Initial stable release.
+* Remove internal selection state logic.
```

Comment:
Is this released? If not, it feels like we don't really need to add this. Perhaps there is a way we can bypass the danger file?

Reply:
bypass danger by removing the `.` from ` * Initial stable release.` 😄